src/emulator/hub.ts

```diff
@@ -0,0 +1,158 @@
+import * as http from "http";
+import * as express from "express";
+import * as os from "os";
+import * as fs from "fs";
+import * as path from "path";
+import * as bodyParser from "body-parser";
+
+import * as utils from "../utils";
+import * as logger from "../logger";
+import { Constants } from "./constants";
+import { Emulators, EmulatorInstance, EmulatorInfo, IMPORT_EXPORT_EMULATORS } from "./types";
+import { HubExport } from "./hubExport";
+
+// We use the CLI version from package.json
+const pkg = require("../../package.json");
```

Comment:
Just double checking: Is this actually packed when we do CLI release? I'm also concerned on how this plays with firepit...

Reply:
I stole this trick from elsewhere in the codebase (all --debug logs include the CLI version) so yeah it is packed.